parquet/src/test/java/org/apache/iceberg/parquet/TestDictionaryRowGroupFilter.java

```diff
@@ -102,21 +103,21 @@
     TOO_LONG_FOR_STATS = sb.toString();
   }
 
-  private static final File PARQUET_FILE = new File("/tmp/stats-row-group-filter-test.parquet");
-  private static MessageType PARQUET_SCHEMA = null;
-  private static BlockMetaData ROW_GROUP_METADATA = null;
-  private static DictionaryPageReadStore DICTIONARY_STORE = null;
+  private static final File parquetFile = new File("/tmp/stats-row-group-filter-test.parquet");
+  private static MessageType parquetSchema = null;
+  private static BlockMetaData rowGroupMetadata = null;
+  private static DictionaryPageReadStore dictionaryStore = null;
```

Comment:
Wasn't all caps the right format for `private static` variables?

Reply:
I think they should be final then as well. I'll check.

Reply:
Yes:
```
[ant:checkstyle] [ERROR] /Users/fokkodriesprong/Desktop/incubator-iceberg/parquet/src/test/java/org/apache/iceberg/parquet/TestDictionaryRowGroupFilter.java:107:30: Name 'PARQUET_SCHEMA' must match pattern '^[a-z][a-zA-Z0-9]*$'. [StaticVariableName]
[ant:checkstyle] [ERROR] /Users/fokkodriesprong/Desktop/incubator-iceberg/parquet/src/test/java/org/apache/iceberg/parquet/TestDictionaryRowGroupFilter.java:108:32: Name 'ROW_GROUP_METADATA' must match pattern '^[a-z][a-zA-Z0-9]*$'. [StaticVariableName]
[ant:checkstyle] [ERROR] /Users/fokkodriesprong/Desktop/incubator-iceberg/parquet/src/test/java/org/apache/iceberg/parquet/TestDictionaryRowGroupFilter.java:109:42: Name 'DICTIONARY_STORE' must match pattern '^[a-z][a-zA-Z0-9]*$'. [StaticVariableName]
```

Reply:
I did revert the change for the `PARQUET_FILE`, since that one is final.